main.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"artist"
+	"db"
+	"register"
+
+	"github.com/jmoiron/sqlx"
+
+	"github.com/gin-gonic/gin"
+
+	_ "github.com/go-sql-driver/mysql"
+)
+
+func main() {
+	sqlDb := db.NewSql()
+	defer sqlDb.Close()
+
+	r := setupRouter(sqlDb)
+
+	_ = r.Run(":8080")
+}
+
+func setupRouter(sqlDb *sqlx.DB) *gin.Engine {
+	router := gin.Default()
+
+	registerRepo := register.New(sqlDb)
+	artistRepo := artist.New(sqlDb)
+
+	router.POST("/api/v1/register", registerRepo.AddUser)
+	router.POST("/api/v1/register/admin_register", registerRepo.AddAdmin)
```

Comment:
`AddUser` or `AddAdmin` name is not suitable for `register` API. Rename it to more relative names